woocommerce/wc-admin
# Update to Gutenberg 3.3 – remove `gutenberg` dependency

Fixes #218 – This PR actually removes the gutenberg package dependency in favor of the newly released sub-packages. This also updates the version of babel used to translate the code, and fixes the places where packages have changed: `utils` has been split apart, so now we import from `keycodes` or `html-entities` directly; `compose` has moved into its own `compose` package.

I needed to update the Jest config and webpack config, and honestly I'm not sure I did this totally correct - especially the jest configs.

**To test**

- Check over the configs, make sure it looks coherent
- Run `npm install` to update your packages
- Run `npm run build` to build the files
- Run `npm start` to try watching files
- Run `npm test` to make sure tests still work
